speed up gffutils

jkniehaus opened the pull request:
added 'disable_infer_genes' and 'disable_infer_transcripts' to 'create_db'
I'm not sure if this will be problematic under different scenarios, but this sped up my peaks2utr command by about 10-fold, from 6 hours to ~35 minutes.
This is my first pull request, and I'm not too familiar w/ python, so I apologize if anything looks wrong/this was too small for a pull request.

Cheers!

haessar commented:
See https://github.com/haessar/peaks2utr/commit/511319e2f50cdd69548526eb0241b7bfeb1319ea

Commit:
Expose gffutils inference options

jkniehaus commented:
Sorry I haven't had a chance to look at this.
I updated this PR against the current master since we're a couple versions behind.

The test you made looked great. I added a fourth criteria for disable_infer*=False with a complete nested .gtf file just to be sure.
I also added `--disable-infer-genes` and `--disable-infer-transcripts`, as CLI options, both defaulting to `False`
These get passed to `gffutils.create_db` and made sure this worked with a mocked test using `python -m pythest -q`.
